modules/nw-customize-ingress-error-pages.adoc

```diff
@@ -0,0 +1,51 @@
+// Module filename: nw-customize-ingress-error-pages.adoc
+// Module included in the following assemblies:
+// * networking/ingress-controller-configuration.adoc
+
+[id="nw-customize-ingress-error-pages_{context}"]
+= Customizing Ingress Controller error pages
+
+As a cluster administrator, you can use the `httpErrorCodePages` field to specify custom error code pages for the Ingress Controller. {product-title} router serves a 503 error page by default when the server is not running or when the route is incorrect or non-existent. When you specify a custom error code page, {product-title} serves either a 503 or 404 error page when the requested URL does not exist. The default HTTP 503 error page is not served.
+
+See the link:https://raw.githubusercontent.com/openshift/router/master/images/router/haproxy/conf/error-page-503.http[default 503 error page].
+
+.Procedure
+
+. Create a configmap named "my-custom-error-code-pages" in the `openshift-config` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-config create configmap my-custom-error-code-pages
+--from-file=error-page-503.http
+--from-file=error-page-404.http
+----
++
+[NOTE]
+====
+The configmap keys have two available formats, error-page-503.http and error-page-404.http. You can use one or both of the available formats.
```

Comment:
I moved this info to the introduction paragraph. I removed the note since I relocated the information; however, I can add it back and repeat the key file names.